lib/cea/cea608_data_channel.js

```diff
@@ -462,26 +487,27 @@ shaka.cea.Cea608DataChannel = class {
           shaka.cea.Cea608Memory.CharSet.PORTUGUESE_GERMAN:
           shaka.cea.Cea608Memory.CharSet.SPANISH_FRENCH;
 
-    this.curbuf.addChar(charSet, b2);
+    this.curbuf_.addChar(charSet, b2);
   }
 
   /**
    * Decodes control code.
    * Three types of control codes:
    * Preamble Address Codes, Mid-Row Codes, and Miscellaneous Control Codes.
-   * @param {!shaka.cea.Cea608DataChannel.CcData} ccPacket
+   * @param {!shaka.cea.CeaDecoder.CCPacket} ccPacket
```

Comment:
This is still technically a reference to the decoder, which may bite us later.  Can this type move back to this namespace?

Reply:
I understand the concern with having this refer back to the CeaDecoder. However, I realized that the CEA-708 decoder will also use this struct, and it doesn't make full sense having it in this CEA-608-specific file. It should be in a more general file, which I thought would be the CeaDecoder (which at some point will decode both 608 and 708).

Maybe I should move this up to the decoder interface instead, and both the concrete decoder as well as this data channel can refer to it from there?